bootstrap.py

```diff
@@ -88,21 +88,27 @@ def choose_color_theme(window):
 
 '''})
     view.set_syntax_file('Packages/MarkdownEditing/Markdown.tmLanguage')
-    default_mde_scheme = sublime.load_settings('Markdown.sublime-settings').get('color_scheme') or 'Packages/MarkdownEditing/MarkdownEditor.tmTheme'
+    default_mde_scheme = sublime.load_settings('Markdown.sublime-settings').get('color_scheme') or 'Packages/MarkdownEditing/MarkdownEditor.sublime-color-scheme'
     print(default_mde_scheme)
     view.settings().set('color_scheme', default_mde_scheme)
     view.set_read_only(True)
     view.set_scratch(True)
 
     global_scheme = sublime.load_settings('Preferences.sublime-settings').get('color_scheme')
-    themes = ['Packages/MarkdownEditing/MarkdownEditor.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Focus.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Yellow.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-ArcDark.tmTheme',
+    themes = ['Packages/MarkdownEditing/MarkdownEditor.sublime-color-scheme',
+              'Packages/MarkdownEditing/MarkdownEditor-Focus.sublime-color-scheme',
+              'Packages/MarkdownEditing/MarkdownEditor-Yellow.sublime-color-scheme',
+              'Packages/MarkdownEditing/MarkdownEditor-Dark.sublime-color-scheme',
+              'Packages/MarkdownEditing/MarkdownEditor-ArcDark.sublime-color-scheme',
               global_scheme]
 
-    themes_display = [re.search('[^/]+(?=\.tmTheme$)', s).group(0) + (' (Current)' if s == default_mde_scheme else '') + (' (Global)' if s == global_scheme else '') for s in themes]
+    themes_display = []
+    for s in themes:
+        m = re.search('[^/]+(?=\\.sublime-color-scheme$)', s)
+        if m is None:
+            continue
+        theme_display = m.group(0) + (' (Current)' if s == default_mde_scheme else '') + (' (Global)' if s == global_scheme else '')
```

Comment:
Lines should not exceed 120 chars. Bad code style.

Probably don't need to force 80 chars, but 90 or 100 is enough.